Dockerfile_archive

```diff
@@ -12,28 +12,32 @@ RUN apt-get update --yes \
         iproute2 \
  && apt-get upgrade --yes \
  && apt-get satisfy --yes --no-install-recommends \
+       ## Standard packages
         ca-certificates \
         cargo \
-        dpkg-dev \
         gcc \
         git \
         libclang-dev \
+       #libgpgme-dev \
         libpython3-dev \
         libssl-dev \
         libtdb-dev \
         make \
         mypy-protobuf \
         pkg-config \
         protobuf-compiler \
-        python3-apt \
         python3-gpg \
         python3-pip \
         python3-setuptools \
-        python3-setuptools-protobuf \
         python3-setuptools-rust \
         python3-wheel \
         python3-wheel-whl \
         rustc \
+       #swig \
```

Comment:
Why is swig added here?

Reply:
Trying to have a 'standard' template for Dockerfiles_*

This gets cleaned up in another commit that hasn't yet been cherry picked

Reply:
Commit it - please review again